apisix/plugins/forward-auth.lua

```diff
@@ -27,6 +27,12 @@ local schema = {
             type = "boolean",
             default = true,
         },
+        request_method = {
+            type = "string",
+            default = "GET",
+            enum = {"GET", "POST"},
```

Comment:
How about the `PUT` method?

Reply:
I don't think we need it. it maybe a burden for the user. `GET & POST` is enough